pkg/server/commands/reverseexpand/reverse_expand.go

```diff
@@ -474,9 +474,7 @@ func (c *ReverseExpandQuery) readTuplesAndExecute(
 	// filter out invalid tuples yielded by the database iterator
 	filteredIter := storage.NewFilteredTupleKeyIterator(
 		storage.NewTupleKeyIteratorFromTupleIterator(iter),
-		func(tupleKey *openfgav1.TupleKey) bool {
-			return validation.ValidateCondition(c.typesystem, tupleKey) == nil
-		},
+		validation.FilterInvalidTuples(c.typesystem),
```

Comment:
(suggestion): Would this make more sense to have on the tuple package? `tuple.FilterInvalid`

Validation should be a func that's provide for a given package IMO. 

No need to change this now, but something to keep in mind as we slowly improve our package design.

Reply:
Yes, it would make more sense in `tuple` but i won't change it now